### PR TITLE
Refactor ex01

### PR DESCRIPTION
**Pull request by barashit**
Add self-descriptive variables and descriptive text

### CODE REVIEW

File: chapter02/exercises/ex01/main.cpp [hunk 1]
```diff
@@ -10,7 +10,11 @@
 int main()
 {
     std::string address = "Gantiadi, 24";
-    std::string name = "Nadia Sitkareva";
+    std::string first_name = "Nadia";
```
Comment (dino-source) [inline chapter02/exercises/ex01/main.cpp]: Please consider to use 'user' prefix before your variables, as foolows:
user_first_name
user_last_name
user_address

**Review by dino-source — CHANGES_REQUESTED** [review]

File: chapter02/exercises/ex01/main.cpp (hunk 1 above)
Comment (barashit) [reply chapter02/exercises/ex01/main.cpp]: done

**Review by dino-source — DISMISSED** [review]
Well done!

**Review by dino-source — APPROVED** [review]
Well done. Everything looks great to me.

**Pull request merged**